### PR TITLE
Improve CI diagnosability

### PR DESCRIPTION
**Pull request by rainersigwald**
I was looking at enabling CI for full-framework builds in xplat and noticed again that we don't reliably fail the script on intermediate step failures.  This has caused some confusion in the past and it's clearly just a bad idea.

### CODE REVIEW

File: BuildAndCopy.cmd [hunk 1]
```diff
@@ -25,14 +25,25 @@ echo ** Additional Build Parameters:%AdditionalBuildCommand%
...
 :: Build MSBuild
 call "%~dp0build.cmd" /t:Rebuild %AdditionalBuildCommand%
+set BUILDERRORLEVEL=%ERRORLEVEL%
 
 :: Kill Roslyn, which may have handles open to files we want
 taskkill /F /IM vbcscompiler.exe
 
+if %BUILDERRORLEVEL% NEQ 0 (
+    echo.
+    echo Failed to build with errorlevel %BUILDERRORLEVEL% 1>&2
```
Comment (cdmihai) [inline BuildAndCopy.cmd]: Since there's three runs, would it be useful to also echo the current log file?

Comment (rainersigwald) [inline BuildAndCopy.cmd]: What do you mean "current log file"?  Something like "For more details see %MSBUILDLOGPATH%"?

Comment (cdmihai) [inline BuildAndCopy.cmd]: Yes. It may cut down the time required to figure out which build failed.

Comment (rainersigwald) [inline BuildAndCopy.cmd]: I added the log path to the last message from `build.cmd` (the common script).  That work?

**Comment by rainersigwald** [comment]
Looks like this will continue to fail until the update to `netci.groovy` is pushed.  Does it look ok, @cdmihai?

**Comment by cdmihai** [comment]
Yup, It should work :)
OSX and linux just produce "msbuild.log" for now, but that will get captured by "msbuild*log"

**Pull request merged**